### ISSUE TITLE
Deprecate csv geometry examples and data

### ISSUE THREAD

**Issue by beomki-yeo** [acts-project/traccc#661]
I don't think they are useful anymore in traccc.
I suggest to cleanup the README instructions, relavant data and I/O codes.
@krasznaa @stephenswat How do you think?

**Comment by stephenswat** [comment]
I think we still need the CSV data, but I agree that the CSV geometries can go.

**Comment by beomki-yeo** [comment]
Yeah I was meaning the csv geometries only and data generated with csv geometries

**Comment by krasznaa** [comment]
:thinking: Let's talk about this today. As I've thought about this a fair amount wrt. #627. Removing the ability to read CSV geometry files is not going to get rid of nearly as much code as you might think. But it's easier if we discuss this in person...
